src/client/components/Dashboard.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import DeckItem from './DeckItem';
+// import _ from 'underscore';
+import $ from 'jquery';
```

Comment:
Sounds good. I didn't realize it was a materialize dep. If so then I'm cool to use it here.